src/napari_time_slicer/_function.py

```diff
@@ -52,27 +54,23 @@ def convert_to_stack4d(layer : LayerInput, viewer: napari.Viewer) -> Layer:
         _set_timepoint(viewer, f)
 
         # get the layer data at a specific time point
-        result_single_frame = np.asarray(layer.data).copy()
-
-        if len(result_single_frame.shape) == 2:
-            result_single_frame = np.asarray([result_single_frame])
+        result_single_frame = deepcopy(layer.data)
```

Comment:
Why do we need to copy here?

Reply:
@haesleinhuepf Good question...not sure why I built it in there. I think I may have been  afraid of accidentally overwriting data in the source layer, but it's probably not an issue here. I removed it in ace0796a1a6325308c315d8f0a93dbffeee16da8. Tests are still passing and it also worked interactively 👍